packages/lodestar/src/node/nodejs.ts

```diff
@@ -193,11 +196,57 @@ export class BeaconNode {
       api,
     });
 
-    await network.start();
-    await sync.start();
-    chores.start();
+    // await network.start();
+    // await sync.start();
+    // chores.start();
 
-    void runNodeNotifier({network, chain, sync, config, logger, signal});
+    // void runNodeNotifier({network, chain, sync, config, logger, signal});
+
+    await libp2p.start();
+    setTimeout(async function () {
+      for (let i = 0; ; i++) {
+        const toDialPeers: PeerId[] = [];
+        for (const peer of libp2p.peerStore.peers.values()) {
+          if (!libp2p.connectionManager.get(peer.id)) {
+            toDialPeers.push(peer.id);
+          }
+          if (toDialPeers.length >= 25) break;
+        }
+        logger.info("Run libp2p.dial", {i, toDialPeers: toDialPeers.length});
+        for (const peer of toDialPeers) {
```

Comment:
this is same code I found in https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar/src/network/peers/discover.ts#L153